pkg/configv1/application.go

```diff
@@ -59,6 +59,7 @@ type GenericApplicationSpec struct {
 	// Configuration for drift detection
 	DriftDetection *DriftDetection `json:"driftDetection"`
 	// List of the plugin name
+	// This field is plugin-specific, so intentionally restrict the access for the actual value here and decode it on the SDK side.
```

Comment:
So we should rewrite the above line either. Since it's not just list of plugin name anymore

Reply:
Thanks, fixed [2dba71d](https://github.com/pipe-cd/pipecd/pull/5801/commits/2dba71d0adfc3efe0a6fa796193073d1c216816f)